evm/src/main/java/org/hyperledger/besu/evm/operation/JumpiOperation.java

```diff
@@ -20,9 +20,6 @@
 import org.hyperledger.besu.evm.frame.MessageFrame;
 import org.hyperledger.besu.evm.gascalculator.GasCalculator;
 
-import java.util.Optional;
-import java.util.OptionalLong;
-
 import org.apache.tuweni.bytes.Bytes;
 
 public class JumpiOperation extends AbstractFixedCostOperation {
```

Comment:
Was there not a similar benefit to a static OperationResult(10,null) success response here?

Reply:
no, there wasn't.  I was suprised.